web/plugin-manager/src/styles/GlobalStyles.tsx

```diff
@@ -0,0 +1,18 @@
+import { createGlobalStyle } from "styled-components";
```

Comment:
I don't think it is a good practise to use several ui framework, daisyUI or styledComponent, you should use only one.

Reply:
Daisy Ui is a component library , we pick only what we want from this , it's not a framework to for  styledComponent it's mandatory for the rebranding. 

Styled Component will be use for Twin Macro to have a clear code . it will increase robustness

So we will need this :)